services/core-api/app/api/mines/permits/permit_amendment/resources/permit_amendment_issue_vc.py

```diff
@@ -0,0 +1,33 @@
+import decimal
```

Comment:
Looks like this file contains a lot of unused imports, can we clean them up?

Reply:
done